pkg/apis/stable/v1alpha1/fleetautoscaler.go

```diff
@@ -91,6 +100,22 @@ type BufferPolicy struct {
 	BufferSize intstr.IntOrString `json:"bufferSize"`
 }
 
+// WebhookPolicy controls the desired behavior of the webhook policy.
+type WebhookPolicy struct {
+	Service *AutoscalerService `json:"service"`
+	URL     string             `json:"url"`
+}
+
+// AutoscalerService is the description of the webhook autoscaler service
+// used to form url which is accessible inside the cluster
+type AutoscalerService struct {
+	Name string `json:"name"`
```

Comment:
Why don't you embed this type ? https://github.com/kubernetes/api/blob/173ce66c1e39d1d0f56e0b3347ff2988068aecd0/admissionregistration/v1beta1/types.go#L247

I saw your comment in examples/webhookfleetautoscaler.yaml I figured you would use it.

Reply:
+1 - we should embed the existing types, in this case WebhookClientConfig

Reply:
Totally agree with you